src/jvm/test/resources/expected_logs/trace_debugger/failing_nextInt_trace_debugger.txt

```diff
@@ -15,17 +15,3 @@ The following interleaving leads to the error:
 | ---------------------------------------- |
 | operation(): IllegalArgumentException #1 |
 | ---------------------------------------- |
-
-Exception stack traces:
-#1: java.lang.IllegalArgumentException: Random range is empty: [1, 1).
-	at kotlin.random.RandomKt.checkRangeBounds(Random.kt:378)
-	at kotlin.random.Random.nextInt(Random.kt:65)
-	at org.jetbrains.kotlinx.lincheck_test.representation.trace_debugger.FailingCustomRandomCallRepresentationTest.operation(RandomRepresentationTests.kt:40)
-	at java.base/java.lang.Thread.run(Thread.java:840)
```

Comment:
Would it be possible to keep the exception stack trace in this (and similar) case?